tasks/managed/rh-sign-image-cosign/rh-sign-image-cosign.yaml

```diff
@@ -147,7 +148,8 @@ spec:
           fi
 
           if [ "$found_signatures" -eq 0 ]; then
-            run_cosign -t 3m0s sign "$COSIGN_REKOR_ARGS" \
+            # shellcheck disable=SC2086
```

Comment:
Same as above, why is this needed?

Reply:
fixed

Reply:
It's not correct right now. An array needs to be used. Right now it's just one big string.